aztec/src/main/kotlin/org/wordpress/aztec/formatting/InlineFormatter.kt

```diff
@@ -176,6 +185,15 @@ class InlineFormatter(editor: AztecText, val codeStyle: CodeStyle) : AztecFormat
         joinStyleSpans(start, end)
     }
 
+    private fun applyMarkInlineStyle(start: Int = selectionStart, end: Int = selectionEnd, attrs: AztecAttributes = AztecAttributes()) {
+        val previousSpans = editableText.getSpans(start, end, MarkSpan::class.java)
+        previousSpans.forEach {
+            (it as MarkSpan).applyInlineStyleAttributes(editableText, start, end)
```

Comment:
I think we could omit the casting here (`it as MarkSpan`) since `previousSpans` contains an array of `MarkSpan!`